workspaces/bulk-import/plugins/bulk-import-backend/src/schema/openapi.yaml

```
@@ -360,6 +361,38 @@ components:
       schema:
         type: integer
         default: 1
+    sortColumnQueryParam:
+      in: query
+      name: sortColumn
+      description: |
+        The allowed values for sorting columns:
+        - `repository.name`: Sort by repository name.
+        - `repository.organization`: Sort by organization URL.
+        - `repository.url`: Sort by repository URL.
+        - `lastUpdate`: Sort by the last time the catalog-info.yaml was updated.
+        - `status`: Sort by the status of the catalog-info.yaml.
+      required: false
+      schema:
+        enum:
```

Please add type also here:
```suggestion
      schema:
        type: string
        enum:
```